data/pages.yaml

```diff
@@ -53,6 +49,56 @@ novice:
   learn: Understand the fundamentals of Ansible automation
   install: Install the Ansible package
   start: Write your first playbook in a few easy steps
+# Resources for user journeys
+user:
+  title: User resources
+  intro: Automate the management of remote systems and control their desired state.
+  milestone:
+    automate: Create automation
+    build_inventory: Build inventories
+    organize: Organize automation projects
+    tooling: Use Ansible tooling
+    reuse: Re-use automation content
+    share: Share automation content
+    awx: Schedule and run automation jobs
+    executor: Embed automation in systems
+  automate:
+    intro: Write playbooks and develop content that automates complex tasks.
+    start: Start writing Ansible playbooks
+    evaluate_plugins: Learn which plugins to use for your tasks
+  build_inventory:
+    intro: Build inventories that list and describe your infrastructure.
+    multiple_hosts: Build inventory files to manage multiple hosts
+    dynamic: Use dynamic inventories
+  organize:
+    intro: Organize your automation project as it grows.
+    roles: Use roles to structure the automation project
+  tooling:
+    intro: Validate and improve your project for consistent, trusted automation.
+    lint: Use Ansible Lint to validate playbooks
```

Comment:
Consider removing all the intros - I think they add to the wall of text. And each link has it's own description so I'm not sure we gain much with the intros.

Reply:
I'll do that in a separate issue so we only have to do it once without rebasing.